devicon.json

```
@@ -7509,6 +7509,29 @@
         "color": "#2298BD",
         "aliases": []
     },
+    {
+        "name": "tauri",
+        "altnames": [],
+        "tags": [
+            "rust",
+            "electron",
+            "desktop"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "color": "#000",
```

The `color` attribute should be the main/primary color of the icon, unless something else is specified in the Branding Guidelines. In the case of `tauri`, page 12 of their [branding guidelines](https://tauri.app/assets/files/Brand_Guidelines-9aae083028ca7fc274a201f83bc082b3.pdf#12) specify that the primary color is `#FFC131 ` 
```suggestion
        "color": "#FFC131",
```